src/main/java/org/idea/plugin/atg/psi/reference/contribution/RepositoryXmlContributor.java

```diff
@@ -74,6 +74,13 @@ public void registerReferenceProviders(@NotNull PsiReferenceRegistrar registrar)
                                         .andOr(XmlPatterns.xmlTag().withParent(XmlPatterns.xmlTag().withName(Repository.ITEM_DESCRIPTOR)),
                                                 XmlPatterns.xmlTag().withSuperParent(2, XmlPatterns.xmlTag().withName(Repository.ITEM_DESCRIPTOR))))),
                 new ItemDescriptorReferenceProvider());
+
+        registrar.registerReferenceProvider(XmlPatterns.xmlAttributeValue()
+                        .withParent(XmlPatterns.xmlAttribute().withName(Repository.ITEM_TYPE)
```

Comment:
withName allows to check as one of names,
so item-type could be added to org/idea/plugin/atg/psi/reference/contribution/RepositoryXmlContributor.java:72